### DISCUSSION TITLE
error when make liburing2.6

### DISCUSSION THREAD

**Discussion by 970315420** [axboe/liburing#1202] [Q&A]
When I make liburing2.6 ,the error occurs as below
`In file included from send-zerocopy.c:15:
/usr/include/linux/ipv6.h:19:8: error: redefinition of ‘struct in6_pktinfo’
   19 | struct in6_pktinfo {
      |        ^~~~~~~~~~~
In file included from /usr/include/arpa/inet.h:22,
                 from send-zerocopy.c:13:
/usr/include/netinet/in.h:538:8: note: originally defined here
  538 | struct in6_pktinfo
      |        ^~~~~~~~~~~
/usr/include/linux/ipv6.h:24:8: error: redefinition of ‘struct ip6_mtuinfo’
   24 | struct ip6_mtuinfo {
      |        ^~~~~~~~~~~
/usr/include/netinet/in.h:545:8: note: originally defined here
  545 | struct ip6_mtuinfo
      |        ^~~~~~~~~~~
make[1]: *** [send-zerocopy.t] Error 1
`
Why does this happen?
ps:
kernel version 5.1
gcc version 12

**Comment by axboe** [comment]
This is fixed in the current tree, for a while.
